nifi-nar-bundles/nifi-stateful-analysis-bundle/nifi-stateful-analysis-processors/src/test/java/org/apache/nifi/processors/stateful/analysis/TestAttributeRollingWindow.java

```diff
@@ -192,81 +192,4 @@ public void testVerifyCount() throws InterruptedException {
 
     }
 
-
-    @Test
-    public void testMicroBatching() throws InterruptedException {
```

Comment:
Can we add ``@Ignore`` instead? Reading unit test code is, sometimes, useful for people to understand how the processor is supposed to work.

Reply:
That and a //TODO: investigate would be good.